### PR TITLE
Add workOrderId field for work order Error response from enclave

### PR DESCRIPTION
**Pull request by manju956**
Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: listener/avalon_listener/tcs_work_order_handler.py [hunk 1]
```diff
@@ -131,14 +131,18 @@ def WorkOrderGetResult(self, **params):
                 err_code = response["error"]["code"]
                 err_msg = response["error"]["message"]
 
+                data = {
```
Comment (Ram-srini) [inline listener/avalon_listener/tcs_work_order_handler.py]: This is taken care in PR#367

File: tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp [hunk 2]
```diff
@@ -435,6 +435,9 @@ namespace tcf {
         JsonSetStr(resp, "jsonrpc", "2.0", "failed to serialize jsonrpc");
         JsonSetNumber(resp, "id", json_request_id, "failed to serialize json request id");
 
+        JsonSetStr(resp, "workOrderId", work_order_id.c_str(),
```
Comment (Ram-srini) [inline tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp]: "workOrderId" should be inside "data". It should be like
{
"jsonrpc": "2.0",
"id": <jrpc_request_id>,
"error": {
"code": <error_code>,
"message": <error_message>,
"data": {
"workOrderId": <work_order_id
}
}

Comment (manju956) [reply tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp]: Done

File: listener/avalon_listener/tcs_work_order_handler.py (hunk 1 above)
Comment (manju956) [reply listener/avalon_listener/tcs_work_order_handler.py]: I shall remove this once PR#367 is merged.

File: tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp [hunk 3]
```diff
@@ -444,6 +444,8 @@ namespace tcf {
 
         JsonSetNumber(error, "code", err_code, "failed to serialize error code");
         JsonSetStr(error, "message", err_message, "failed to serialize error message");
+        JsonSetStr(error, "workOrderId", work_order_id.c_str(),
```
Comment (Ram-srini) [inline tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp]: workOrderId should be inside data field as below
"data": {
"workOrderId": <work_order_d>
}

Comment (manju956) [reply tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp]: Ok, will make the correction

**Pull request merged**